libs/designsystem/src/lib/components/item/item.component.integration.spec.ts

```diff
@@ -1,177 +1,64 @@
-        <kirby-list [items]="[{ name: 'Item1' }, { name: 'Item2' }, { name: 'Item3' }]">
-          <kirby-card *kirbyListItemTemplate="let item">
-            <kirby-item>{{ item.name }}</kirby-item>
-          </kirby-card>
-        </kirby-list>
-        `
-      );
-      ionList = spectator.queryHost('ion-list');
-      await TestHelper.whenReady(ionList);
-      itemsInList = spectator.queryAll('ion-list ion-item');
-    });
-
-    it('should create list wrapper', () => {
-      expect(spectator.component).toBeTruthy();
-    });
-
-    it('should render items in list', () => {
-      expect(itemsInList).not.toBeEmpty();
-      itemsInList.forEach((item) => {
-        expect(item.shadowRoot.hasChildNodes()).toBeTrue();
-      });
-    });
-
-    it('should render first and last item without padding top/bottom', async () => {
-      const firstItem = itemsInList[0].shadowRoot.querySelector('.item-native');
-      const lastItem = itemsInList[itemsInList.length - 1].shadowRoot.querySelector('.item-native');
-      expect(firstItem).toHaveComputedStyle({ 'padding-top': '0px' });
-      expect(lastItem).toHaveComputedStyle({ 'padding-bottom': '0px' });
-    });
-  });
-});
+import { createHostFactory, SpectatorHost } from '@ngneat/spectator';
+
+import { DesignTokenHelper } from '@kirbydesign/core';
+
+import { IconComponent } from '..';
+import { TestHelper } from '../../testing/test-helper';
+
+import { LabelComponent } from '.';
+import { ItemComponent } from './item.component';
+
+const { fontWeight } = DesignTokenHelper;
+
+describe('ItemComponent', () => {
+  let ionItem: HTMLElement;
+
+  let spectator: SpectatorHost<ItemComponent>;
+  const createHost = createHostFactory({
+    component: ItemComponent,
+    imports: [TestHelper.ionicModuleForTest],
+    declarations: [LabelComponent, IconComponent],
+  });
+
+  describe('selectable and selected with kirby-label', () => {
+    let labelElements: Element[];
+
+    beforeEach(async () => {
+      spectator = createHost<ItemComponent>(
+        `
+        <kirby-item selectable="true" selected="true">
+          <kirby-label>
+            <h3>Title</h3>
+            <p detail>Detail</p>
+          </kirby-label>
+          <kirby-label slot="end">
+            <data>Value</data>
+            <data detail>Detail</data>
+          </kirby-label>
+        </kirby-item>
+        `
+      );
+      ionItem = spectator.queryHost('ion-label');
+      await TestHelper.whenReady(ionItem);
+      labelElements = spectator.queryAll(
+        'ion-item ion-label > :is(h1, h2, h3, h4, h5, h6, p, data)'
+      );
+    });
+
+    it('should render general header, data and paragraph elements in bold', () => {
+      labelElements
+        .filter((e) => !e.attributes.getNamedItem('detail'))
+        .forEach((e) => {
+          expect(e).toHaveComputedStyle({ 'font-weight': fontWeight('bold') });
+        });
+    });
+
+    it('should render detail data and paragraph elements in default normal', () => {
```

Comment:
Same as above comment regarding use of values in test descriptions

Reply:
Done